### PR TITLE
Solution Branch

### PR DESCRIPTION
**Pull request by beckybeauchamp1**
I pushed up the solution branch, please let me know if I should add or change anything!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,13 +16,13 @@ the square brackets (for multiple-choice questions).
...
-[] Strings
-[] Booleans
-[] Undefined
+[X] Strings
+[X] Booleans
+[X] Undefined
 [] NaN
-[] Integers
+[X] Integers
 [] Arrays
```
Comment (RobertAKARobin) [inline README.md:24]: Arrays are!

Comment (RobertAKARobin) [inline README.md:24]: ...not. :(

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures

Shame on me.

Comment (RobertAKARobin) [inline README.md:24]: Javascript is stupid.

**Comment by RobertAKARobin** [comment]
:+1:

**Comment by jshawl** [comment]
:+1: except this pr _should not_ be merged to master

instead:

```
git push origin :solution # delete solution branch
git branch -m solution becky-solution # rename your solution
git push origin master:solution # create solution branch based off master
git push origin becky-solution # push your solution branch
submit pr to solution
```

**Pull request closed without merging**